Packages/com.unity.inputsystem/InputSystem/InputSettings.cs

```diff
@@ -717,21 +717,6 @@ public void SetInternalFeatureFlag(string featureName, bool enabled)
             if (string.IsNullOrEmpty(featureName))
                 throw new ArgumentNullException(nameof(featureName));
 
-            switch (featureName)
```

Comment:
Slightly confused about this code removal as it seemed unused

Reply:
Same here

Reply:
They were used by various other classes via internal scope access